crates/typst/src/meta/mod.rs

```diff
@@ -0,0 +1,15 @@
+use crate::doc::{Lang, Region};
+use crate::model::StyleChain;
+
+/// The named with which an element is referenced.
+pub trait LocalName {
```

Comment:
This trait isn't worth a new top-level module. You could move it to the bottom of model/element.rs.

Reply:
Ok sure